<launch>
    <node name="planner_avc" pkg="rr_common" type="planner" output="screen" >
        <param name="N_PATH_SEGMENTS" value="3"/>
        <param name="N_CONTROL_SAMPLES" value="300"/>
        <param name="SMOOTHING_ARRAY_SIZE" value="20"/>
        <param name="SEGMENT_DISTANCES" value="2 3 3"/>
        <param name="STEER_LIMITS" value="0.35 0.35 0.35"/>
        <param name="STEER_STDDEVS" value="0.22 0.16 0.15"/>
        <param name="INPUT_CLOUD_TOPIC" value="/scan/pointcloud"/>
        <param name="MAX_SPEED" value="2.0"/>
        <param name="PATH_SIMILARITY_CUTOFF" value="0.20"/>
        <param name="MAX_RELATIVE_COST" value="7.0"/>

        <param name="WHEEL_BASE" value="1.17"/>
        <param name="COLLISION_RADIUS" value="0.70"/>
    </node>
</launch>
